src/cli/cli.cc

```diff
@@ -47,11 +47,11 @@ StatusCode Install(AkliteClient &client, int version) {
   }
 
   TufTarget target;
-  if (version == -1) {
+  if (version == -1 && target_name.empty()) {
     target = cr.GetLatest();
   } else {
     for (const auto &t : cr.Targets()) {
-      if (t.Version() == version) {
+      if (t.Version() == version || t.Name() == target_name) {
```

Comment:
I think its fine to continue fighting the version number/name thing like this. Eventually we probably have to build some kind of "version" object that can abstract this.